docs/developing-fuzzbench/adding_a_new_benchmark.md

```diff
@@ -35,22 +35,57 @@ powerful but also more work.
 
 ## OSS-Fuzz benchmarks
 
-You can use any existing OSS-Fuzz project as a benchmark. You just need to
-create a single file `oss-fuzz.yaml` in the benchmark directory.
+You can use most existing OSS-Fuzz projects a benchmark. First find out which
+project and the fuzz target you want to use as a benchmark. Then find out the commit 
+from the project repo which you want to use. Finally, find out the date and time 
+(UTC) of that commit in iso format. This can be done in the project repo as follows:
+```shell
+git --no-pager log -1 $COMMIT_HASH --format=%cd --date=iso-strict
+```
 
-```yaml
-project: <project-name>
-fuzz_target: <fuzz-target-name>
-oss_fuzz_builder_hash: <sha>
+Once you have this information, run
```

Comment:
Doesn't need to come now, but we should probably walkthrough the process of getting this info.
For example, we could show how to get this info to make benchmark for the "foo" fuzz target of LLVM using the latest commit.